doc/classes/PopupMenu.xml

```diff
@@ -256,12 +279,14 @@
 			<argument index="0" name="idx" type="int">
 			</argument>
 			<description>
+				Returns the text [String] associated with the specified [code]idx[/code].
 			</description>
 		</method>
 		<method name="is_hide_on_window_lose_focus" qualifiers="const">
 			<return type="bool">
 			</return>
 			<description>
+				Returns whether the popup will be hidden when the window loses focus or not.
```

Comment:
Iirc the guideline for methods that return booleans is to write `Returns [code]true[/code] if...`
Suggestion: `Returns [code]true[/code] if the PopupMenu is set to hide when it loses focus`.